content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -286,7 +286,7 @@ workloads running in a shared cluster. Running workloads in a sandbox environmen
 insulate the host from container escapes, where an attacker exploits a vulnerability to gain
 access to the host system and all the processes/files running on that host.
 
-Virtual machines and userspace kernels are 2 popular approaches to sandboxing. The following
+Virtual machines and userspace kernels are two popular approaches to sandboxing. The following
```

Comment:
Seems like a trivial change to the docs, @mikejoh can you find more of the changes that can be made in this doc so that it can be improved further.

Reply:
Not in this document AFAICT!